src/components/toolbox/withdraw/Withdraw.tsx

```diff
@@ -34,10 +34,7 @@ interface WithdrawFormValues {
 }
 
 const WITHDRAW_GAS = 55000;
-const DVM_WITHDRAW_ADDRESS = '0x0000000000000000000000000000000000000015';
-const EMPTY_ADDRESS = '0x0000000000000000000000000000000000000000000000000000000000000000';
-
-const registry = new TypeRegistry();
+const DVM_DISPATCH_ADDRESS = '0x0000000000000000000000000000000000000019';
```

Comment:
Add todo comment here: https://github.com/darwinia-network/darwinia-common/pull/1298

Reply:
Which network is supported?

Crab? or Crab and Pangolin?